src/services/news/index.js

```diff
@@ -148,6 +163,10 @@ class NewsService {
 		const news = await this.app.service('newsModel').get(id);
 		this.checkExistence(news, id);
 		await this.authorize(news, params.account, 'NEWS_VIEW');
+		await this.getPermissions(news, params.account, {
+			NEWS_VIEW: 'view',
+			NEWS_EDIT: 'edit',
```

Comment:
you should probably add `NEWS_CREATE: 'delete'` here for completeness. At the moment, NEWS_EDIT and NEWS_CREATE (which covers both creation and deletion rights) are only ever given simultaneously (to the same role), so edit==create==delete,but this could change. cc: @konstantin-sc, @CeEv

Reply:
Existing permissions in file services are delete, create, read(=view) and write(=edit). It is also contain that every user has full access to the content that is created by his self. Than it make sense to split the create and delete permissions. Every role x can create something. But only admins can delete content from other persons.

Reply:
Yes! I've added a ticket.